code/mob/living/critter/ai/roach.dm

```diff
@@ -0,0 +1,4 @@
+/datum/aiHolder/roach
+	New()
+		..()
+		default_task = get_instance(/datum/aiTask/timed/wander, list(src))
```

Comment:
I'm biased because I just reviewed the mouse one but can we make roaches eat things too?

Reply:
Roaches can eat, as a treat